src/store/data_persistence/address.ts

```diff
@@ -1,12 +1,11 @@
 import { DataPersistenceMutationType } from '@src/view_models/DataPersistence';
 import { mutation } from '@src/store/util';
-import { SET_ADDRESS_FIELD } from '@src/store/address/mutationTypes';
 
 export default ( namespace: string ) => {
 	return {
 		storageKey: 'address',
 		mutationType: DataPersistenceMutationType.KEY_VALUE_PAIR,
-		mutationKey: mutation( namespace, SET_ADDRESS_FIELD ),
+		mutationKey: mutation( namespace, 'SET_ADDRESS_FIELD' ),
 		fields: [
 			'salutation',
 			'title',
```

Comment:
wondering how that ever properly worked before..

Reply:
The `SET_ADDRESS_FIELD` constant was a string `"SET_ADDRESS_FIELD"`, so this is it inlined